inlong-audit/audit-sdk/src/main/java/org/apache/inlong/audit/util/AuditManagerUtils.java

```diff
@@ -76,6 +86,89 @@ public static int buildAuditId(AuditIdEnum baseAuditId,
         return Integer.parseInt(auditPreFix + buildAuditIdSuffix(baseAuditId.getValue()), 2);
     }
 
+    public static AuditInformation buildAuditInformation(String auditType,
+            FlowType dataFlow,
+            boolean success,
+            boolean isRealtime,
+            boolean discard,
+            boolean retry) {
+        String auditPreFix = buildSuccessAndFailureFlag(success) +
+                buildRealtimeFlag(isRealtime) +
+                buildDiscardFlag(discard) +
+                buildRetryFlag(retry);
+        AuditIdEnum baseAuditId = AuditIdEnum.getAuditId(auditType, dataFlow);
+        int auditId = Integer.parseInt(auditPreFix + buildAuditIdSuffix(baseAuditId.getValue()), 2);
+        StringBuilder nameInEnglish = new StringBuilder()
+                .append(baseAuditId.getAuditType().value())
+                .append(AUDIT_DESCRIPTION_JOINER)
+                .append(dataFlow.getNameInEnglish())
+                .append(AUDIT_DESCRIPTION_JOINER);
+        StringBuilder nameInChinese = new StringBuilder()
+                .append(baseAuditId.getAuditType().value())
+                .append(dataFlow.getNameInChinese());
+
+        if (discard) {
+            nameInEnglish.append("discard").append(AUDIT_DESCRIPTION_JOINER);
+            nameInChinese.append("丢弃");
+        }
+        if (retry) {
+            nameInEnglish.append("retry").append(AUDIT_DESCRIPTION_JOINER);
+            nameInChinese.append("重试");
+        }
+        if (success) {
+            nameInEnglish.append("success");
+            nameInChinese.append("成功");
+        } else {
+            nameInEnglish.append("failed");
+            nameInChinese.append("失败");
+        }
+        if (!isRealtime) {
+            nameInEnglish.append("(CheckPoint)");
```

Comment:
Why checkpoint need brackets?

Reply:
CheckPoint is placed at the end in parentheses to facilitate the distinction between real-time audit and checkpoint audit.